build/params_2k.go

```diff
@@ -53,8 +52,6 @@ var UpgradeChocolateHeight = abi.ChainEpoch(-17)
 var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
 var UpgradeSkyrHeight = abi.ChainEpoch(100)
 
-var ActorsCIDs = map[actors.Version]cid.Cid{}
```

Comment:
We're now including this information in the `build/builtin_actors_gen.go` file. Unfortunately, we need the ability to _switch_ bundles at runtime (for testing) so we can't actually guarantee a single CID.